utils/SnapPy/Snappy/EEMEP/Controller.py

```diff
@@ -91,20 +91,24 @@ def __init__(self):
         self.lastQDict = {}
         self.lastLog = ""
 
-    def write_log(self, txt:str):
-        if (txt != self.lastLog):
-            self.lastLog = txt
-            debug(txt)
-            self.main.evaluate_javaScript('updateEemepLog({0});'.format(json.dumps(txt)))
+    def write_log(self, txt:str, clear_log=False):
```

Comment:
The clear_log flag is used to clear the log explicitly. Otherwise, it keeps the log from previously.